pkg/apis/build/v1alpha1/build_pod.go

```diff
@@ -84,6 +88,21 @@ var (
 		MountPath: "/builderPullSecrets",
 		ReadOnly:  true,
 	}
+	notaryV1Volume = corev1.VolumeMount{
+		Name:      notaryDirName,
+		MountPath: "/var/notary/v1",
+		ReadOnly:  true,
+	}
+	reportVolume = corev1.VolumeMount{
```

Comment:
Could report.toml just be written to the /layers mount?

Reply:
No, not exactly sure what export does to the layers volume, but post export, that volume attempts to execute the launcher when mounted.

Reply:
Ahh as in mounting `/layers` overwrites the completion binary?

Reply:
Ah yup, thats exactly right.